### PR TITLE
adding starter file for scss

### PR DESCRIPTION
**Pull request by salviosage**
#### What does this PR do?

- add sass for global component 

#### Description of Task to be completed?

- sass for components,   layout, themes, toolbar, toast, spinners, global CSS 

#### How should this be manually tested?
clone repo and run sass --watch UI/css/main.scss UI/css/main.css in a command line to watch any changes and compiles all scss to CSS 

#### What are the relevant pivotal tracker stories?

- #167298313

#### Screenshots (if appropriate)
image

#### Questions:

### CODE REVIEW

**Comment by nshutijonathan** [comment]
You can please check the PR name so that It can follow recommended naming convention, otherwise, that's a good job

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
PR naming convention, please. You forgot story id

**Comment by Anguandia** [comment]
Hi, there's no scss file, so the test command can not run in terminal, please check it up

**Review by minega25 — CHANGES_REQUESTED** [review]
@salviosage This is amazing work. I like the way you have structured things. However the branch name format does not confirm to the Andela standard, and more comments to your code can help us

File: UI/templates/index.html [hunk 1]
```diff
@@ -7,8 +7,36 @@
     <title>AndaApp</title>
     <link rel="stylesheet" href="../css/general.css">
     <link rel="stylesheet" href="../css/custom.css">
+    <link rel="stylesheet" href="../css/main.css">
```
Comment (nkalyesubula) [inline UI/templates/index.html:10]: Would you move all the necessary code that is in main.css  to general.css

File: UI/css/main.css [hunk 2]
```diff
@@ -0,0 +1,276 @@
+/*
+$primary-color-dark:   #388E3C;
+$primary-color:        #4CAF50;
+$primary-color-light:  #C8E6C9;
+$primary-color-text:   #FFFFFF;
+$accent-color:         #FF4081;
+$primary-text-color:   #212121;
+$secondary-text-color: #727272;
+$divider-color:        #B6B6B6;
+*/
```
Comment (nkalyesubula) [inline UI/css/main.css:10]: Would you remove all the commented code?

File: UI/css/_global.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
+@font-face {
```
Comment (nkalyesubula) [inline UI/css/_global.scss:1]: Would you kindly add a profound description to each section of the code?

**Review by nkalyesubula — CHANGES_REQUESTED** [review]
Thank you for your hard work. Would you rename your branch to follow the right naming convention? Secondly, the pivotal tracker story id included in your  PR description should be a clickable link that takes you to the pivotal tracker board where that feature or chore is defined.  Lastly, I think is this a chore, not a feature. thanks

**Review by habinezadalvan — COMMENTED** [review]
This is a great work @salviosage, I really appreciate it. Would you mind to add some comments so that we can clearly understand your codes?

**Review by habinezadalvan — APPROVED** [review]

**Pull request closed without merging**